app/views/contacts/_sidebar.html.haml

```diff
@@ -6,9 +6,14 @@
   .text-center.spotlight
     = link_to t("contacts.spotlight.community_spotlight"), community_spotlight_path, class: "btn btn-link"
 .text-center
-  .btn.btn-link{ "data-toggle" => "modal", "data-target" => "#invitationsModal"}
+  #invitations-button.btn.btn-link{ "data-toggle" => "modal" }
     = t("invitations.new.invite_someone_to_join")
   = render "shared/modal",
       path: new_user_invitation_path,
       id: "invitationsModal",
       title: t("invitations.new.invite_someone_to_join")
+
+:javascript
+  $("#invitations-button").click(function(){
+    app.helpers.showModal("#invitationsModal");
+  });
```

Comment:
I'd like to. I haven't been able to make this work in `app/assets/javascripts/app/views/contact_stream_view.js` nor in `app/assets/javascripts/app/views/contact_view.js`.